### PR TITLE
Added youtube link

### PR DESCRIPTION
**Pull request by kaugsrha**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -21,8 +21,13 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0)
     eventmachine (1.2.7)
+<<<<<<< HEAD
```
Comment (dfridovi) [inline Gemfile.lock:24]: fix

File: index.html [hunk 2]
```diff
@@ -17,7 +17,8 @@
         <span style="display: block; margin-bottom: 3em"></span>
 
       <a href="https://github.com/CLeARoboticsLab"><i class="fa fa-github"></i> CLeAR Lab</a>&nbsp;&nbsp;&nbsp;
-      <a href="mailto:dfk@utexeas.edu"><i class="fa fa-envelope-o"></i> dfk@utexas.edu</a>
+      <a href="mailto:dfk@utexeas.edu"><i class="fa fa-envelope-o"></i> dfk@utexas.edu</a>&nbsp;&nbsp;&nbsp;
```
Comment (dfridovi) [inline index.html:20]: typo fix

**Review by dfridovi — CHANGES_REQUESTED** [review]
Guessing you hadn't pulled latest develop when you started, and ended up with some merge issues. See comments.

File: index.html (hunk 2 above)
Comment (kaugsrha) [reply index.html:20]: I didn't get what the typo is?

**Comment by dfridovi** [comment]
Read the email address againSent from my iPhoneOn Sep 25, 2023, at 9:27 PM, Kushagra Gupta ***@***.***> wrote:﻿
@kaugsrha commented on this pull request.



In index.html:
> @@ -17,7 +17,8 @@
         <span style="display: block; margin-bottom: 3em"></span>
 
       <a href="https://github.com/CLeARoboticsLab"><i class="fa fa-github"></i> CLeAR Lab</a>&nbsp;&nbsp;&nbsp;
-      <a ***@***.***"><i class="fa fa-envelope-o"></i> ***@***.***</a>
+      <a ***@***.***"><i class="fa fa-envelope-o"></i> ***@***.***</a>&nbsp;&nbsp;&nbsp;

I didn't get what the typo is?

**Comment by dfridovi** [comment]
You didn’t add the typo.. it must have been there before but now is as good a time to fix it as anySent from my iPhoneOn Sep 25, 2023, at 9:56 PM, David Fridovich-Keil ***@***.***> wrote:﻿Read the email address againSent from my iPhoneOn Sep 25, 2023, at 9:27 PM, Kushagra Gupta ***@***.***> wrote:﻿
@kaugsrha commented on this pull request.



In index.html:
> @@ -17,7 +17,8 @@
         <span style="display: block; margin-bottom: 3em"></span>
 
       <a href="https://github.com/CLeARoboticsLab"><i class="fa fa-github"></i> CLeAR Lab</a>&nbsp;&nbsp;&nbsp;
-      <a ***@***.***"><i class="fa fa-envelope-o"></i> ***@***.***</a>
+      <a ***@***.***"><i class="fa fa-envelope-o"></i> ***@***.***</a>&nbsp;&nbsp;&nbsp;

I didn't get what the typo is?

**Review by dfridovi — APPROVED** [review]

**Pull request merged**